src/components/BulkEditList/BulkEditList.js

```diff
@@ -195,7 +198,7 @@ export const BulkEditList = () => {
           defaultWidth="300px"
           paneTitle={<FormattedMessage id="ui-bulk-edit.list.criteriaTitle" />}
         >
-          <BulkEditListSidebar />
+          <BulkEditListSidebar isBuildQueryButtonDisabled={isBuildQueryButtonDisabled} />
```

Comment:
Please remove this prop. Bulk edut list sidebar should not know anything about disabling of button on specific tab. Please call `  const { bulkDetails } = useBulkOperationDetails({ id: bulkOperationId, additionalQueryKeys: [step] });
` directly inside QueryTab

Reply:
done